### ISSUE TITLE
API returns `id` in attributes of webform entity and breaks json-api

### ISSUE THREAD

**Issue by adborden** [18F/beta.foia.gov#197]
The API returns `id` in the attributes of a webform entity, but according to [json-api spec](http://jsonapi.org/format/#document-resource-object-fields), this is not allowed. The json-api parsing library we're using loses the form id which we need for submission. This might be a bug in the json-api drupal model.

We've worked around the issue in the front stage, so not a blocker.

**Comment by malikkotob** [comment]
@adborden you're right. Good catch. This may be the fault of the `webform` module. I'm not quite sure. Even if it's not a blocker, thanks for opening the issue. Something to look into.

**Comment by adborden** [comment]
Iceboxing as this is not critical.
